tools/omero/omero_dataset_to_plate.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="omero_dataset_to_plate" name="OMERO Dataset to Plate" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
+   <description> with omero-py </description>
+   <macros>
+       <import>macros.xml</import>
+       <token name="@VERSION_SUFFIX@">3</token>
+   </macros>
+   <xrefs>
+       <xref type="bio.tools">omero</xref>
+   </xrefs>
+   <expand macro="omeropy_requirements"/>
+   <command detect_errors="exit_code"><![CDATA[
+       python '$__tool_directory__/omero_dataset_to_plate.py'
+       --credential-file '$credentials'
+       @HOST_PORT@
+       --dataset_id '$dataset_id'
+       --log_file '$log'
+       --mapping_file '$mapping'
+       --delete_dataset '$delete_dataset'
+   ]]></command>
+   <configfiles>
+       <expand macro="credentials"/>
+   </configfiles>
+   <inputs>
+       <expand macro="host_port"/>
+       <param name="dataset_id" type="integer" optional="false" label="Dataset ID to convert to a plate"/>
+       <param name="mapping" type="data" format= "tabular" optional="false" label="Mapping file"/>
+       <param name="delete_dataset" type="boolean" truevalue="True" falsevalue="False" checked="false" label="Delete Dataset" help="Delete dataset after import"/>
```

Comment:
What exactly is deleted? The image dataset in omero?

Reply:
This will delete the original dataset from where the plate is created from. 
Basically, images are copied into in the new plate format and then the full dataset is deleted.